### ISSUE TITLE
Convert scripts to entrypoints

### ISSUE THREAD

**Issue by jhunkeler** [spacetelescope/jwst#4105]
We should review the status of these scripts and consolidate the useful ones back into their parent pipeline step. I propose doing it like this `jwst/[step_name]/cli.py` or this  `jwst/[step_name]/cli/name_here.py` so it will be possible to generate `entry_points` in `setup.py`  instead of curating them all by hand.

## Scripts
```bash 
$ grep -E -l -r 'argv\[.*\]|(opt|arg)parse' *                                                                                                                                                                          
jwst/associations/lib/lv2_v1_to_v2.py
jwst/associations/main.py
jwst/associations/asn_from_list.py
jwst/datamodels/schema_editor.py
jwst/exp_to_source/main.py
jwst/csv_tools/csvconvert.py
jwst/ramp_fitting/mc_3d.py
jwst/ramp_fitting/mc_2d.py
jwst/ramp_fitting/create_cube.py
jwst/ramp_fitting/rotate_dataset.py
jwst/ramp_fitting/compare_cr_navg_files.py
jwst/ramp_fitting/compare_cr_files.py
jwst/ramp_fitting/compare_crs.py
jwst/stpipe/cmdline.py
scripts/strun
scripts/utc_to_bary.py
scripts/jump
scripts/cube_build
scripts/create_data
scripts/straylight
scripts/set_velocity_aberration.py
scripts/assign_wcs
scripts/outlier_detection
scripts/stspec
scripts/linearitycorr
scripts/asn_edit
scripts/dark_current
scripts/persistencecorr
scripts/resample
scripts/data_generate
scripts/collect_pipeline_cfgs
scripts/set_bary_helio_times.py
scripts/okify_regression_tests.py
scripts/rampfitcorr
scripts/ipc
scripts/move_wcs.py
scripts/flatfieldcorr
scripts/make_header
scripts/set_telescope_pointing.py
scripts/schemadoc
scripts/world_coords
scripts/photomcorr
scripts/coron
```

**Comment by stscijgbot-jp** [comment]
_This issue is tracked on JIRA as [JP-1616](https://jira.stsci.edu/browse/JP-1616)._

**Comment by jdavies-st** [comment]
Resolved by #7406
